curriculum/dictionaries/spanish/comments.js

```diff
@@ -0,0 +1,454 @@
+  },
+  {
+    id: '6rztdg',
+    text: 'Agrega un registro aquí'
+  },
+  {
+    id: 'to1vwe',
+    text: 'Agrega los métodos handleChange() y submitMessage() aquí'
+  },
+  {
+    id: '31b7ey',
+    text: 'Agrega tu código encima de esta línea'
+  },
+  {
+    id: 'c24by8',
+    text: 'Agrega tu código debajo de esta línea'
+  },
+  {
+    id: 'jbrt8k',
+    text: 'Agrega la fila número m a newArray'
+  },
+  {
+    id: 'zkh12d',
+    text:
+      'Después de pasar el desafío experimenta con myString y observa cómo funciona la agrupación'
+  },
+  {
+    id: 'mobihi',
+    text: 'Asigna 5 a oopsGlobal aquí'
+  },
+  {
+    id: 'v3ups9',
+    text: 'Llama scale con un argumento aquí'
+  },
+  {
+    id: 'iw4a3a',
+    text: 'Caso 1: El objetivo no tiene hijo'
+  },
+  {
+    id: '463xp8',
+    text: 'Caso 2: El objetivo tiene un hijo'
+  },
+  {
+    id: 'u3inrm',
+    text: 'caso 3: El objetivo tiene dos hijos'
+  },
+  {
+    id: 'axnbgg',
+    text: 'Cambia el código encima de esta línea'
+  },
+  {
+    id: 'i2kck7',
+    text: 'Cambia el código debajo de esta línea'
+  },
+  {
+    id: 'dlbobn',
+    text: 'Cambia esta línea'
+  },
+  {
+    id: 'v127zb',
+    text:
+      'Verifica las dos consolas para ver la diferencia. La consola de freeCodeCamp debería haber impreso la variable dos veces, una por cada prueba de este desafío. La consola del navegador solo debería imprimir la variable una vez dado que la limpiaste primero.'
+  },
+  {
+    id: 'ejm0ql',
+    text: 'Cierra la tercera pestaña en videoWindow y une'
+  },
+  {
+    id: 'iwch6t',
+    text: 'Completa el método de abajo e implementa los otros de manera similar'
+  },
+  {
+    id: 'hihhyz',
+    text: 'Completa la sentencia return:'
+  },
+  {
+    id: 'sdxti5',
+    text: 'Cuenta los hijos del objetivo a eliminar'
+  },
+  {
+    id: 'wfw6sc',
+    text: 'Crea una escala de X y Y'
+  },
+  {
+    id: 'sjw6f4',
+    text: 'Crea la escala aquí'
+  },
+  {
+    id: 'nupsh2',
+    text: 'Crea un arreglo de 2 dimensiones con m filas y n columnas de ceros'
+  },
+  {
+    id: 'xfjb3s',
+    text: 'Declara la variable myGlobal debajo de esta línea'
+  },
+  {
+    id: 'htpjk7',
+    text: 'Define una constante para acciones de decremento'
+  },
+  {
+    id: 'tfzdsp',
+    text: 'Define una constante para acciones de incremento'
+  },
+  {
+    id: 'zh20mi',
+    text: 'Define ADD, addMessage(), messageReducer(), y store aquí:'
+  },
+  {
+    id: '43qs4c',
+    text: 'Define un creador de acción para decrementar'
+  },
+  {
+    id: 'nen3qo',
+    text: 'Define un creador de acción para incrementar'
+  },
+  {
+    id: '0cwyam',
+    text: 'Define un creador de acción aquí:'
+  },
+  {
+    id: 'fq0wsg',
+    text: 'Define una acción aquí:'
+  },
+  {
+    id: 'tegkqa',
+    text: 'Define el componente Container aquí:'
+  },
+  {
+    id: 'b5oihn',
+    text:
+      'Define el contador reductor que aumentará o disminuirá el estado en función de la acción que reciba'
```

Comment:
The English text is:
> Define the counter reducer which will increment or decrement the state based on the action it receives

The word `counter` should not be translated as it represents a code variable.